credentials/apps/api/v2/serializers.py

```diff
@@ -322,29 +323,35 @@ class SignatoryListField(serializers.ListField):
     Populate signatories images in dict files instead of strings with their filenames.
     """
 
-    def to_internal_value(self, data: str) -> Dict[str, Union[str, InMemoryUploadedFile]]:
+    def to_representation(self, data):
+        return super().to_representation(data.all())
+
+    def to_internal_value(self, data: List[str]) -> List[Optional[Dict[str, Union[str, InMemoryUploadedFile]]]]:
         """
         Converts a list of dict that is a string to a list of dict.
         """
-        data_ = self.populate_signatories(data, self.context["request"].FILES)  # pylint: disable=no-member
-        return super().to_internal_value(data_)
+        try:
+            jsonified_data = list(map(ast.literal_eval, copy(data)))
+        except ValueError:
+            return []
+        else:
+            data_ = self.populate_signatories(
```

Comment:
Please fix this. Use `_data` instead of `data_`

Reply:
Corrected